node-src/ui/messages/errors/unparseableConfigurationFile.ts

```diff
@@ -3,9 +3,11 @@ import { dedent } from 'ts-dedent';
 
 import { error } from '../../components/icons';
 
-export const unparseableConfigurationFile = (configFile: string, err: Error) =>
-  dedent(chalk`
-    ${error} Configuration file {bold ${configFile}} could not be parsed, is it valid JSON?
+export const unparseableConfigurationFile = (configFile: string, err: Error) => {
```

Comment:
Should we add stories for the new file extensions?

Reply:
Yeah, we can I suppose. I didn't change much other than just the wording `JSON` to `JSON` or `JSON5`, depending on the extension.

Reply:
Done!